### PR TITLE
refactor: dark mode tweaks for pagination

### PR DESCRIPTION
**Pull request by Highjhacker**
## Summary

Tweaks to follow new dark theme design for pagination.

## Checklist

- [ ] Documentation _(if necessary)_
- [ ] Tests _(if necessary)_
- [x] Ready to be merged

### CODE REVIEW

File: resources/views/pagination-url.blade.php [hunk 1]
```diff
@@ -18,12 +18,12 @@
 
         @if($paginator->onFirstPage())
             <div class="flex items-center button-generic button-disabled">
-                <div class="flex items-center">@svg('chevron-left', 'h-3 w-3')<span class="hidden lg:flex lg:ml-2">Previous</span></div>
+                <div class="flex items-center"><span class="hidden lg:flex lg:ml-2">Previous</span></div>
```
Comment (ItsANameToo) [inline resources/views/pagination-url.blade.php:21]: don't remove that, this pagination component is used by more projects and removing that will need to be discussed first

Comment (ItsANameToo) [reply resources/views/pagination-url.blade.php:21]: also it's unrelated to dark mode

Comment (Highjhacker) [reply resources/views/pagination-url.blade.php:21]: Alrighty, gonna put them back, just followed the designs ; https://gyazo.com/e44e2a2cdbeecc6213d3f0a60624f4c9

Comment (alexbarnsley) [reply resources/views/pagination-url.blade.php]: @Highjhacker I think the chevron you put back might be in the wrong place based on the pr diff

**Comment by ItsANameToo** [comment]
same change was made in #34

**Pull request closed without merging**